SchedulePlatform.WebUI/Pages/Services.razor.cs

```diff
@@ -16,7 +21,10 @@ protected override async Task OnInitializedAsync()
             var json = await response.Content.ReadAsStringAsync();
 
             services = JsonConvert.DeserializeObject<List<ServiceProvided>>(json);
+
         }
     }
+
+    
```

Comment:
I'm a bit confused about the empty new lines added in here. I think they shouldn't be here.